internal/flink/command_artifact.go

```diff
@@ -13,11 +13,12 @@ type flinkArtifactOut struct {
 	Id            string `human:"ID" serialized:"id"`
 	Name          string `human:"Name" serialized:"name"`
 	Version       string `human:"Version" serialized:"version"`
-	Class         string `human:"Class" serialized:"class"`
```

Comment:
This command is currently not GA, right?

If it is GA, then we can't remove fields yet.

Reply:
Thats correct, its not GA yet.